bootstrap/sql/migrations/native/1.3.0/mysql/postDataMigrationSQLScript.sql

```diff
@@ -5,4 +5,45 @@ SET json = REPLACE(json, '"customMetricsProfile"', '"customMetrics"');
 -- Delete customMetricsProfile from entity_extension
 -- This was not supported on the processing side before 1.3.
 DELETE FROM openmetadata_db.entity_extension ee   
-where extension  like '%customMetrics';
\ No newline at end of file
+where extension  like '%customMetrics';
+
+-- BEGIN: Incident Manager Migration
+-- STEP 1: Update test case testCaseResult.testCaseFailureStatus field
+UPDATE test_case tc
```

Comment:
should we just delete any existing references to resolution instead of migrating?

Reply:
We can, if we do so should then not migrate anything and just deleted existing reference?

Reply:
thats what I am thinking, so that we don't do any big migration. IMO not sure how many users may be using it. If you not heard from any user in community support or customers. My suggestion is to drop everything related to resolution and bring the new functionality

Reply:
I have not. Will do